auth/requirements.txt

```diff
@@ -1,4 +1,8 @@
 aiohttp >= 2.0.0, < 4.0.0
 backoff >= 1.0.0, < 2.0.0
 cryptography >= 2.0.0, < 3.0.0
+future >= 0.17.0, < 0.18.0
 pyjwt >= 1.5.3, < 2.0.0
+requests >= 2.20.0, < 2.23.0
```

Comment:
I think this `requirements.txt` should align with the usage of file-on-disks, eg. `-aio`, so that we don't break being able to do a local `pip install -e ./` and get things running -- anywhere we add/remove dependencies to do `aio` -> `rest` should be outside of this file.

Another option would be to comment it out in the file-on-disk and uncomment it in the py3to2 script (in the same way as we comment out aiohttp we could uncomment `requests`)

Reply:
Makes sense, updating it.